Add CSS revert-rule keyword (#43808)
mdn/content: files/en-us/web/css/reference/values/data_types/index.md

      - : Rolls back the cascade to the value of the earlier origin.
    - {{CSSXref("revert-layer")}}
      - : Rolls back the cascade to the value of the earlier [cascade layer](/en-US/docs/Web/CSS/Reference/At-rules/@layer).
    - {{CSSXref("revert-rule")}}

>       - : Rolls back the cascade as if the current style rule were not present.

Reviewer: To keep this parallel with other descriptions here ("earlier origin", "earlier cascade layer"), which also helps to instantly bring out the differences across `revert`, `revert-layer`, and `revert-rule`, how about this wording:
```suggestion
      - : Rolls back the cascade to the value of an earlier matching style rule.
```
